courses/free-courses-hi.md

```diff
@@ -618,6 +618,7 @@
 
 ### Swift
 
+* [Swift programming hindi tutorial for beginners](https://www.youtube.com/playlist?list=PLb5R4QC2DtFv3MvfG42Cd5La34Hwj4pY6) - Code Cat
```

Comment:
Playlist check out, I would suggest changing the title to be Uppercase as in the suggestion below.

```suggestion
* [Swift Programming Hindi Tutorial for Beginners](https://www.youtube.com/playlist?list=PLb5R4QC2DtFv3MvfG42Cd5La34Hwj4pY6) - Code Cat
```